app/tools/routes.js

```diff
@@ -0,0 +1,7 @@
+const handlers = require('./handlers');
+
+
+module.exports = [
+  { name: 'list', pattern: '/tools', handler: handlers.list },
+  { name: 'restart', pattern: '/tools/:id/restart', handler: handlers.restart },
```

Comment:
I would rename this param to `:name` as this is the tool name to restart, e.g. `rstudio`.

Reply:
(I'm working on the endpoint to deploy a tool and the endpoint I have in mind is `POST /tools/:name/deploy`. It's nice to see we thought about similar endpoints)

Reply:
Yep